changelog/64533.fixed.md

```diff
@@ -0,0 +1,3 @@
+Added support for AWS session tokens in salt-cloud.
```

Comment:
This is adding a feature, so the file extension needs to be `added` instead of fixed.